google-ads-examples/src/main/java/com/google/ads/googleads/examples/shoppingads/AddPerformanceMaxRetailCampaign.java

```diff
@@ -124,8 +128,11 @@
 
     @Parameter(
         names = ArgumentNames.FINAL_URL,
-        description = "The final URL for the asset group of the campaign.")
-    private String finalUrl = "http://www.example.com";
+        required = true,
+        description =
+            "The final url for the generated ads. Must have the same domain as the Merchant Center"
+                + " account.")
+    private String finalUrl;
```

Comment:
Making this parameter a mandatory one makes sense to me but it does not seem to be in the current version of C# (golden) and PHP. Should we communicate this to all?

Reply:
You are correct. Looks like we missed that for PHP though, I will fix.

Reply:
Thanks!